app/javascript/packages/validated-field/validated-field.tsx

```diff
@@ -78,6 +97,9 @@ function ValidatedField({
 
   return (
     <lg-validated-field ref={fieldRef}>
+      <script type="application/json" className="validated-field__error-strings">
+        {JSON.stringify(getErrorMessages(inputProps.type))}
+      </script>
```

Comment:
is the only way to pass translations to custom elements via embedded scripts like this?

Reply:
I guess that option still has a two-step approach, but at least it's in memory instead of requiring a serialization pass? I don't feel that strongly